t/lib/OpenQA/Test/FullstackUtils.pm

```diff
@@ -150,11 +150,19 @@ sub wait_for_developer_console_contains_log_message {
     my $log_textarea           = $driver->find_element('#log');
     my $log                    = $log_textarea->get_text();
     my $previous_log           = '';
+    my $timeout                = 10;
```

Comment:
Only 10 seconds. This would likely time-out on my local machine. There's even a comment in the code for setting the `$check_interval` conditionally. I would set this to at least 30 seconds for usual commands and 5 or even 10 minutes for 'paused'. (Travis can be really slow sometimes.)